com.beeper.Beeper.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+      <li>Send scheduled messages</li>
+      <li>One Unified inbox</li>
+    </ul>
+    <p>This wrapper is not officially endorsed by the Beeper Team.</p>
+  </description>
+  <screenshots>
+    <screenshot>
+      <caption>Beeper logged in</caption>
+      <image type="source">https://uploads-ssl.webflow.com/5ffc199ed786423eb2569667/648af12deb905c3acb5fe9c7_screenshot%20no%20background.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="3.77.21" date="2023-09-20"/>
+      <description>
+        <p>Forwarding a message when the right panel was open would crash the app</p>
+        <p>HEIC images would not preview correctly</p>
+      </description>
+      <issues>
+        <p>Upgrade Electron version to v26</p>
+      </issues>
+    <release version="3.76.15" date="2023-09-13"/>
+      <description>
+        <p>Added ability to change your email address in general settings </p>
+      </description>
+      <issues>
+        <p>Upgrading was prevented in certain cases due to an error</p>
+      </issues>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
```

Comment:
you can remove all the one marked `none` to simplify things.

Reply:
Alrighty